AerialNavigation/drone_3d_trajectory_following/Quadrotor.py

```diff
@@ -24,6 +23,9 @@ def __init__(self, x=0, y=0, z=0, roll=0, pitch=0, yaw=0, size=0.25, show_animat
         if self.show_animation:
             plt.ion()
             fig = plt.figure()
+            fig.canvas.mpl_connect('key_release_event',
```

Comment:
Could you please add a single line comment for all mpl_connect function like:
    # for stopping simulation with the esc key.